travis/travis_install_nightly

```diff
@@ -19,6 +19,9 @@ if [ "${LINT_CHECK}" != "0" ]; then
     pip install --upgrade --no-deps git+https://github.com/vauxoo/pylint-odoo.git   # To use last version ever
     npm install -g eslint  # Extra package for pylint-odoo plugin
     # Get pylint configuration file of vauxoo
+    if [ ! -d ${HOME}/maintainer-quality-tools/travis/cfg ]; then
+        git clone git@github.com:Vauxoo/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools/
```

Comment:
Are you sure that these lines are the real error?
I mean, we don't have changes since a long time

![zapata](https://cloud.githubusercontent.com/assets/6644187/22746717/6aec7af4-edea-11e6-8f4c-1525f99d01c8.png)

And we had builds without errores after that:
![screen shot 2017-02-08 at 10 43 19 am](https://cloud.githubusercontent.com/assets/6644187/22746791/a0468a6e-edea-11e6-8fea-db0f12177e03.png)